src/components/listings/AlbumListing.tsx

```diff
@@ -0,0 +1,58 @@
+import { format } from 'date-fns'
+import React from 'react'
+import { View, Text, Image, ImageBackground } from 'react-native'
+
+interface AlbumListingProps {
+  title: string
+  dateFrom: Date
+  dateTo: Date
+  images?: Array<typeof Image>
+}
+
+function AlbumListing(props: AlbumListingProps) {
+  const {
+    title,
+    dateFrom = new Date('2020-02-23T00:00:00+0000'),
```

Comment:
I think we should probably remove this default values for dates

Reply:
👍